src/posix/platform/include/openthread/openthread-system.h

```diff
@@ -242,6 +242,44 @@ typedef struct otSysInfraNetIfAddressCounters
  */
 void otSysCountInfraNetifAddresses(otSysInfraNetIfAddressCounters *aAddressCounters);
 
+#if OPENTHREAD_POSIX_CONFIG_MULTITHREAD_SUPPORT_ENABLE
+/**
+ * This function create thread and processing OpenThread stack.
+ *
+ * @param[out]   aInstance    A pointer to the OpenThread instance.
+ * @param[in]    aComPort     A pointer to the radio device name.
+ * @param[in]    aOtLogLevel  Ot log level.
+ *
+ */
+void otSysGetInstance(otInstance **aInstance, const char *aComPort, uint16_t aOtLogLevel);
```

Comment:
Use `otLogLevel aLogLevel` instead of `uint16_t aOtLogLevel`?
Need to `#include <openthread/platform/logging.h>` to get the `otLogLevel` definition.

Reply:
Updated